src/AdoPat/AdoPat.csproj

```diff
@@ -6,14 +6,15 @@
     <RootNamespace>Microsoft.Authentication.AdoPat</RootNamespace>
   </PropertyGroup>
 
-    <ItemGroup>
+  <ItemGroup>
     <!-- Stylecop required items -->
     <AdditionalFiles Include="..\stylecop\stylecop.json" Link="stylecop.json" />
     <Compile Include="..\stylecop\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.4" />
     <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="4.61.3" />
     <PackageReference Include="Microsoft.VisualStudio.Services.Client" Version="19.239.0-preview" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
+    <PackageReference Include="System.Formats.Asn1" Version="6.0.1" />

```

Comment:
What is `Microsoft.VisualStudio.Services.Client` being used for? Is it a test dependency? If so, is it critical that we pin this transitive dependency? If not, could we include a note here so that it doesn't accidentally stay pinned longer than it needs to be?

Reply:
Nvm, we definitely need this package at runtime. It's not a test dependency.

Could we get a comment in here like this?
```xml
<!-- Transitive dependency of Microsoft.VisualStudio.Services.Client temporarily pinned for security reasons. -->
```

Reply:
We certainly can! That's a good callout.

Reply:
This is done!